src/App.test.tsx

```diff
@@ -1,9 +1,10 @@
 import React from "react";
-import { render, screen } from "@testing-library/react";
+import { createRoot } from "react-dom/client";
 import App from "./App";
 
-test("renders learn react link", () => {
-  render(<App />);
-  const linkElement = screen.getByText(/What to watch/i);
-  expect(linkElement).toBeInTheDocument();
+test("render without crashing", () => {
+  const div = document.createElement("div");
+  const root = createRoot(div);
+  root.render(<App />);
+  root.unmount();
```

Comment:
undo these changes @dima-diachkov has fixed the test in his PR. Will be merged as soon as comments are fixed

Reply:
pulled the master branch